### PR TITLE
[debian] Update changelog for new release 19.12

### PR DESCRIPTION
**Pull request by slashdd**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+      (LP: #829379)
+    - Clear hash id database on package resync. (LP: #1616116)
+    - Permit environments containing unicode chars for script execution.
+      (LP: #1765518)
+    - Set LANG in sysinfo wrapper. (LP: #1780071)
+    - Avoid leaking memory through unused twisted logs. (LP: #1685885)
+    - Handle unicode apt-sources-replace (LP: #1771443)
+    - Add simfs to the whitelist of watched file-systems (LP: #1499104)
+    - Return non-blank message when tasks already performed. (LP: #928933)
+    - Overrid apt progress pulse(). (LP: #1758529)
```
Comment (maxiberta) [inline debian/changelog]: Typo.

Comment (slashdd) [reply debian/changelog]: Fixed in  ea0d4c9.

**Comment by maxiberta** [comment]
Should we add a changelog entry on https://bugs.launchpad.net/ubuntu/+source/landscape-client/+bug/1855522 ?

**Comment by slashdd** [comment]
The fix you are talking about for (LP: #1855522) seems to have been merged upstream after the 19.12 release, thus not in focal as we speak.

**Comment by slashdd** [comment]
```
Tag 19.12:
Date:   Fri Nov 29 09:21:45 2019 -0500

97f74e035 Handle EINVAL error of SIOCETHTOOL ioctl:
Date:   Sat Dec 7 10:11:09 2019 +0100
```

**Pull request closed without merging**